pkg/governance-scripts/contracts/BaseCoordinator.sol

```diff
@@ -37,17 +37,17 @@ interface ICurrentAuthorizer is IAuthorizer {
 }
 
 abstract contract BaseCoordinator is SingletonAuthentication, ReentrancyGuard {
-    IAuthorizerAdaptor private immutable _authorizerAdaptor;
+    IAuthorizerAdaptorEntrypoint private immutable _adaptorEntrypoint;
 
     function()[] private _coordinatorStages;
     uint256[] private _stageActivationTime;
 
-    constructor(IAuthorizerAdaptor authorizerAdaptor) SingletonAuthentication(authorizerAdaptor.getVault()) {
-        _authorizerAdaptor = authorizerAdaptor;
+    constructor(IAuthorizerAdaptorEntrypoint adaptorEntrypoint) SingletonAuthentication(adaptorEntrypoint.getVault()) {
+        _adaptorEntrypoint = adaptorEntrypoint;
     }
 
-    function getAuthorizerAdaptor() public view returns (IAuthorizerAdaptor) {
-        return _authorizerAdaptor;
+    function getAuthorizerAdaptorEntrypoint() public view returns (IAuthorizerAdaptor) {
+        return _adaptorEntrypoint;
```

Comment:
We're expecting this migrator to run prior to the authorizer migration so we don't need to involve the authorizer adaptor entrypoint here.

Reply:
If we want to make this change to the BaseCoordinator this should be done post migration.